### DISCUSSION TITLE
Automatically detecting missing collections

### DISCUSSION THREAD

**Discussion by rob-bc** [meisnate12/Plex-Meta-Manager#301] [Q&A] [answered]
I currently use the [Radarr Collection and People Manager](https://github.com/RhinoRhys/radarr-collections) to automatically check my movies against TMDB Collections and add any missing movies. Is it possible to do a similar thing with this app, so that I don't have to manually tell it which collections I want to create? Apologies if I am missing something obvious or if this is just outside the scope of the tool.

Massive thanks to meisnate12 for all the great work!

**Comment by meisnate12** [comment, marked as answer]
Not currently but it's a planned feature #169 right now you still have to at least tell it what tmdb collections to monitor

**Reply by rob-bc** [reply]
Ah thanks for pointing that out - I missed that! Ok well I'll work with that then for now :)
